script/webpack-bundle-analyzer.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * hack to run the analyzer as a webpack plugin taken from:
+ * https://github.com/facebook/create-react-app/issues/3518
+ */
+
+process.env.NODE_ENV = "production"
```

Comment:
Ah, concatenating files (& causing the single entry of `index.ts + 54 files (concatenated)` is a default config option for `production`: https://webpack.js.org/plugins/module-concatenation-plugin/ ... so we can still analyze the individual file sizes if dev mode was turned on.

Would it be useful to default to `production` but allow a `NODE_ENV=development yarn analyze-build`?

Reply:
that would be cool, except https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/scripts/build.js#L13